### PR TITLE
Updates to TS05, 06, and Culmination activities

### PR DESCRIPTION
**Pull request by mjones01**
plus minor changes (comments, _include files) in TS00, 02, 03, and 04.

### CODE REVIEW

**Comment by lwasser** [comment]
@mjones01 i am going to assume this is ready to merge. For future - you should be able to reference a commit number in the issues rather than a PR. the commit is actually more useful anyway given it's the actual change where as a PR is just a PR. that way we can assume any PR is always ready to be merged. This is standard GIT practice when you work with others :)

File: TS05-Plotting-Time-Series-ggplot-In-R.Rmd [hunk 1]
```diff
@@ -425,18 +431,40 @@ spaced to improve readability. If we zoom into the plot, all of the bars are
 black.
 
 <div id="challenge" markdown="1">
-##Challenge
+## Challenge
+Without creating a subsetted dataframelot, plot the precipitation data for
```
Comment (lwasser) [inline TS05-Plotting-Time-Series-ggplot-In-R.Rmd:435]: what is a dataframelot, ?? maybe a typo?

File: TS06-Plotting-Facets-And-Group-By-Time-In-R.Rmd [hunk 2]
```diff
@@ -61,14 +61,13 @@ preferably RStudio, loaded on your computer.
...
 {% include/_greyBox-wd-rscript.html %}
-{% include/tutorialSeries/_series_dc-tabular-time-series.html %}
 
 ****
 
 ### Recommended Tutorials
-
-This lessons uses both `dplyr` and `ggplot2`. We recommend the following tutorials
-before working through this one, if you are new to either of the above `R` packages.
+This lessons uses both `dplyr` and `ggplot2`. We recommend the following
```
Comment (lwasser) [inline TS06-Plotting-Facets-And-Group-By-Time-In-R.Rmd:68]: looks like we're still back and forth on "lesson" vs tutorial. Shall we stick with tutorial throughout?

**Comment by lwasser** [comment]
Hi @mjones01 this looks good! There are a few typos and such but all very minor. Given i'm finishing up a few things, i'm going to accept this PR and will fold it into a rebuild with the new metadata lesson which is just about complete. Thank you for submitting this.

File: TS06-Plotting-Facets-And-Group-By-Time-In-R.Rmd (hunk 2 above)
Comment (mjones01) [inline TS06-Plotting-Facets-And-Group-By-Time-In-R.Rmd:68]: Yes.  I've changed everywhere I notice it.  However, I'm still finding myself typing lesson by habit.

File: TS05-Plotting-Time-Series-ggplot-In-R.Rmd (hunk 1 above)
Comment (mjones01) [inline TS05-Plotting-Time-Series-ggplot-In-R.Rmd:435]: typo.  As you've merged, I'm assuming you'll make cahnge

File: TS06-Plotting-Facets-And-Group-By-Time-In-R.Rmd (hunk 2 above)
Comment (lwasser) [inline TS06-Plotting-Facets-And-Group-By-Time-In-R.Rmd:68]: No problem. I do the same thing. Let's keep an eye out for each other's use of the word? I'm editing and rebuilding everything now!

File: TS05-Plotting-Time-Series-ggplot-In-R.Rmd (hunk 1 above)
Comment (lwasser) [inline TS05-Plotting-Time-Series-ggplot-In-R.Rmd:435]: Yup - did you get my latest message about the merge? The edits were minor enough, I'll just take care of it.

**Pull request merged**